assets/scss/_component-examples.scss

```diff
@@ -285,6 +285,10 @@
   }
 }
 
+.modal.show {
+  z-index: 1072;
```

Comment:
IMO it's just for the one in our documentation because we want this navbar on top of everything